File: 0001_0100/0035/0035.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    k = int(f.readline())
    while k:
        k -= 1
        n, m = map(int, f.readline().split())
        q.write(str(19 * m + (n + 239) * (n + 366) // 2) + '\n')

File: 0001_0100/0035/0035.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var k, n, m int
	fmt.Fscan(f, &k)
	for ; k > 0; k-- {
		fmt.Fscan(f, &n, &m)
		fmt.Fprintln(q, 19*m+(n+239)*(n+366)/2)
	}
}
